### PR TITLE
Change dob to zodiac sign & update horoscope api consumption

### PR DESCRIPTION
**Pull request by T-willjr**
## Description of Changes & Issue Tag

- Updated routes 
- Both sign in and register buttons on welcome page route properly 
- Updated horoscope api consumption to receive today and yesterday horoscope
- Today and yesterday horoscope displayed on dashboard page

## Concerns
- Need to implement cache to prevent api calls every time user hits dashboard page 
## What GIF best describes how you feel about this PR?
something

### CODE REVIEW

**Comment by SullyBirashk** [comment]
Great Job getting those routes to work!!!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
-    if !@user.dob
-      redirect_to "/register"
-    end
-    zodiac_sign = "leo"
-    day = "today"
-    @horoscope = HoroscopeFacade.get_today_horoscope(zodiac_sign, day)
-  end
+  def sign_up 
+    user = User.find_by(email: session[:access_token])
+    if !user.zodiac_sign || !user.name
```
Comment (jgrazulis) [inline app/controllers/users_controller.rb:19]: neither here nor there, but I love that there's an attribute called `zodiac_sign`

File: app/services/horoscope_service.rb [hunk 2]
```diff
@@ -1,20 +1,10 @@
 class HoroscopeService
   class << self 
-    def call_today_horoscope(zodiac_sign, day)
-      response = conn.get("/api/v1/horoscopes/find?sign=#{zodiac_sign}&day=#{day}")
-      parse_data(response)
+    def call_today_horoscope(zodiac_sign)
+      response = conn.get("/api/v1/horoscopes/find?sign=#{zodiac_sign}")
+      a = parse_data(response)
```
Comment (jgrazulis) [inline app/services/horoscope_service.rb:5]: a?

**Review by jgrazulis — APPROVED** [review]
Looks great!

**Pull request merged**